lib/Runtime/Library/JavascriptGeneratorFunction.cpp

```diff
@@ -378,8 +378,23 @@ namespace Js
         return false;
     }
 
+    BOOL JavascriptGeneratorFunction::SetAccessors(PropertyId propertyId, Var getter, Var setter, PropertyOperationFlags flags)
+    {
+        if (propertyId == PropertyIds::length)
+        {
+            return this->scriptFunction->SetAccessors(propertyId, getter, setter, flags);
+        }
+
+        return JavascriptFunction::SetAccessors(propertyId, getter, setter, flags);
```

Comment:
I see in `GetAccessors`, we directly call `DynamicObject::` methods for `caller` and `arguments`. Should that be done here as well?